src/v2/Apps/ViewingRoom/Components/ViewingRoomsFeaturedRail.tsx

```diff
@@ -16,19 +18,47 @@ export const ViewingRoomsFeaturedRail: React.FC<ViewingRoomsFeaturedRailProps> =
     })
     .filter(Boolean)
 
-  return featuredViewingRoomsForRail.length > 0 && (
-    <Box>
-      <Sans size="5">Featured</Sans>
+  const carouselItemRender = (
+    {
+      heroImageURL,
+      slug,
+      title,
+      partner,
+      status,
+      distanceToOpen,
+      distanceToClose,
+    },
+    slideIndex
+  ): React.ReactElement => {
+    const tag = getTagProps(status, distanceToOpen, distanceToClose)
+    return (
+      <Flex flexDirection="row">
+        {slideIndex !== 0 && <Spacer ml="15px" />}
+        <Link href={`/viewing-room/${slug}`} key={slug} noUnderline>
+          <MediumCard
+            image={heroImageURL}
+            title={title}
+            subtitle={partner.name}
+            tag={tag}
+          />
+        </Link>
+      </Flex>
+    )
+  }
+
+  return (
+    featuredViewingRoomsForRail.length > 0 && (
```

Comment:
It would read a bit better if this was guarded further up towards the top of the function; the component would then exit early and save future devs some reading time:

```tsx
if (featuredViewingRoomsForRail.length === 0) { 
  return null 
}